task.go

```diff
@@ -116,6 +116,8 @@ func (r *marathonClient) KillApplicationTasks(id string, opts *KillApplicationTa
 //	opts:		KillTaskOpts request payload
 func (r *marathonClient) KillTask(taskID string, opts *KillTaskOpts) (*Task, error) {
 	appName := taskID[0:strings.LastIndex(taskID, ".")]
+	appName = strings.Replace(appName, "_", "/", -1)
```

Comment:
>  Doesn't this mean that a call to CreateApplication() will fail as well if the given app ID is invalid?

Yes, it does. Actually it does now to. But the difference is that for now its done after sending request, I'm proposing to do it on method run instead.

> Aren't we yet missing a test that passes an underscored app ID into KillTask and makes sure we're seeing the proper replacement?

Yes, it was missed. I've changed test case to table driven style, added a fixture and a test case for `TaskKill` for application in group